suites/tipset/rewards_penalties.go

```diff
@@ -208,6 +209,61 @@ func TestMinerRewardsAndPenalties(t *testing.T, factory state.Factories) {
 		td.AssertBalance(builtin.BurntFundsActorAddr, big.Add(halfBalance, gasPenalty))
 	})
 
+	t.Run("insufficient gas to cover return value", func(t *testing.T) {
```

Comment:
blocked on https://github.com/filecoin-project/lotus/pull/163 to generate resource files for this test.

Reply:
I think you linked to the wrong PR. Generally I infer that you're blocked on generating gas values from _some_ implementation, and I'll try to do so for you from go-filecoin.

Reply:
I did, corrected.